src/osd/OSD.cc

```diff
@@ -5942,7 +5943,8 @@ OSDService::ScrubJob::ScrubJob(const spg_t& pg, const utime_t& timestamp, bool m
       sched_time += rand() % (int)(g_conf->osd_scrub_min_interval *
```

Comment:
I would like to introduce new settings only if it is really necessary, in order to not overbloat user interface. That is why I am trying to keep only one scrub interval. What if we use pool_scrub_interval to override both osd_scrub_min_interval and osd_scrub_max_interval, like below?

 if pool_scrub_interval > 0
   sched_time = timestamp + min(pool_scrub_interval, osd_scrub_min_interval)
   deadline = timestamp + max(pool_scrub_interval, osd_scrub_max_interval)

If a user wants scrub to be run for a pool more frequently then for other pools, it sets scrub_interval less than osd_scrub_min_interval, and scrubbing will run more frequently if the OSD is not overloaded. If the user sets scrub_interval more then osd_scrub_max_interval it still will be run with the same frequency as for other pools if the OSD is not overloaded but less frequently if it is.

But if people think it is still confusing and not flexible enough I will add both min and max scrub intervals.